src/avrenderer/vrmanager.h

```diff
@@ -15,6 +15,7 @@ class CVRManager : public IVrManager
 	virtual void runFrame() override;
 	virtual void setVargglesTexture(const vr::Texture_t *pTexture) override;
 	virtual glm::mat4 getHmdFromUniverse() override { return m_hmdFromUniverse; }
+	virtual glm::mat4 getMixedRealityFromUniverse() override { return m_mixedRealityFromUniverse; }
```

Comment:
Why use a dedicated function for this instead of using getUniverseFromOrigin and letting the caller do the invert if they want camera from universe?

Also, now that there's a new origin, gadgets should be able to hook themselves to the MR camera now. That's kinda neat.

Reply:
removed it, i was following hte getHmdFromUniverse pattern, thought maybe it was a caching optimization per frame but either way thats silly for mixed reality, this is only queried one place per frame